client/src/charts/NivoCharts.js

```diff
@@ -608,6 +614,119 @@ NivoResponsiveLine.defaultProps = {
   motion_stiffness: 100,
 };
 
+const BubbleNode = ({ node, style, handlers, theme }) => {
+  if (style.r <= 0) return null;
+
+  const min_node_radius = 2;
+
+  const real_r = node.data.isOuter ?
+    style.r :
+    style.r*node.data.ratio > min_node_radius ?
+      style.r*node.data.ratio :
+      min_node_radius;
+  
+  return (
+    <g transform={`translate(${style.x},${style.y})`}>
+      <circle
+        r={real_r}
+        {...handlers}
+        fill={style.fill ? style.fill : style.color}
+        stroke={style.borderColor}
+        strokeWidth={style.borderWidth}
+      />
+    </g>
+  );
+};
+
+
+export class NivoResponsiveBubble extends React.Component{
+  render(){
+    const{
+      margin,
+      is_money,
+      legends,
+      text_formatter,
+      labelSkipWidth,
+      height,
+      value,
+      name,
+      totalValue,
+      totalName,
+    } = this.props;
+    legends && (legends[0].symbolShape = fixedSymbolShape);
```

Comment:
Mutations :(

Actually, what's going on with `legends` and `fixedSymbolShape`? Why does only the first element of `legens` need it's `symbolShape` property overridden? I don't see `fixedSymbolShape` defined in the PR diff, is this a pattern in NivoCharts?

Reply:
It's a `NivoCharts.js` pattern. (?????????) Will investigate on #500.